### PR TITLE
Cleanup to match latest

### PR DESCRIPTION
**Pull request by tkuhrt**
Signed-off-by: Kuhrt, Tracy A <tracy.a.kuhrt@accenture.com>

### CODE REVIEW

File: common/repositories.sh [hunk 1]
```diff
@@ -177,59 +164,6 @@ transact_repositories=(
...
-  https://github.com/hyperledger/iroha-java.git
-  https://github.com/hyperledger/iroha-javascript.git
-  https://github.com/hyperledger/iroha-python.git
-  https://github.com/hyperledger/iroha-scala.git
-  https://github.com/hyperledger/iroha.git
-)
-
-indy_repositories=(
-  https://github.com/hyperledger/indy-agent.git
-  https://github.com/hyperledger/indy-crypto.git
```
Comment (ryjones) [inline common/repositories.sh]: were the iroha, indy, composer, cello, and quilt stanzas deleted on purpose? I must be missing something

Comment (tkuhrt) [reply common/repositories.sh]: They were duplicated.

Comment (tkuhrt) [reply common/repositories.sh]: Wait...why am I not seeing them anymore

Comment (tkuhrt) [reply common/repositories.sh]: Diff doesn’t show them, but when I look at the file, they are there.

**Pull request merged**